style/base.css

```diff
@@ -3,12 +3,46 @@
   margin: 0;
   padding: 0;
 }
+
+html {
+  font-family: "Roboto", sans-serif;
+  color: rgb(30, 30, 30);
+}
+
 body {
-  padding: 0px 16px;
   width: 100vw;
+  background-color: lavender;
 }
 
 main {
-  display: grid;
-  grid-template-columns: 1fr 1fr 1fr 1fr;
+  /* display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr; */
+  display: flex;
+  flex-direction: column;
+  padding: 0 1rem;
+}
+
+p {
+  margin: 1rem 0;
+  font-size: 1rem;
+  font-weight: 400;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.2rem;
+}
+
+.profile-header {
+  display: flex;
+  justify-items: center;
+  align-items: center;
+}
+
+.profile-header li {
+  list-style-type: none;
+}
+
+.profile-header li img {
```

Comment:
this is quiet complicated. We try to make it as simple as possible. Maybe a class Attribute for the image tag is a better option.